### ISSUE TITLE
Trivial proofs?

### ISSUE THREAD

**Issue by clayrat** [sbp/idris-bi#18]
Should we bring back trivial proofs, i.e. ones provable with a single `Refl`? I've been leaving them out as they're useless as lemmas due to Idris's strong normalization, but maybe they're still useful as simple sanity checks?

**Comment by sbp** [comment]
Yeah, I wouldn't mind having them around. Since they're trivial they shouldn't slow down the type checker, and they are valid theorems being proven. I thought there was some other reason why you took them out, but if it's just because they can't be relied upon because of the strong norming, then I think simple sanity checking is always good.
